pkg/volume/cephfs/cephfs.go

```diff
@@ -390,22 +382,11 @@ func (cephfsVolume *cephfs) execFuseMount(mountpoint string) error {
 		keyringFile = cephfsVolume.secretFile
 	}
 
-	// build src like mon1:6789,mon2:6789,mon3:6789:/
```

Comment:
You may need keep the comments.

Reply:
yes, please keep the comments, they're useful.